charts/api/templates/_helpers.tpl

```diff
@@ -88,10 +88,12 @@ Common lists of environment variables
 - name: WBSTACK_ELASTICSEARCH_ENABLED_BY_DEFAULT
   value: {{ .Values.wbstack.elasticSearch.enabledByDefault | quote }}
 {{- end }}
-{{- if .Values.trustedProxy.proxies }}
 - name: TRUSTED_PROXY_PROXIES
-  value: {{ join "," .Values.trustedProxy.proxies | quote }}
-{{- end }}
+  valueFrom:
```

Comment:
didn't think about the with the previous patchset but perhaps we could reduce the scope of this to only the pods that need it? I think this is only the ones that are proxied to (e.g. not backend, or the queue etc.). OTOH it doesn't really hurt to have this everywhere?

Reply:
Yeah good point. Theoretically this should only be needed by `web` as it's the only thing that interfaces with the outside world. What's the situation like for MediaWiki?